matplotlibrc.template

```diff
@@ -642,7 +642,7 @@
 #savefig.facecolor : white       ## figure facecolor when saving
 #savefig.edgecolor : white       ## figure edgecolor when saving
 #savefig.format    : png         ## png, ps, pdf, svg
```

Comment:
Should this also be changed to `{png, ps, pdf, svg}`?

Also these? 
- Line 357: `gridlines at major, minor or both ticks` -> `gridlines at {major, minor, both} ticks`
- Line 359: `left, right and center` -> `{left, right and center}`

Reply:
Good idea. Thanks!

Reply:
@timhoffm you missed the suggested fix at line 644 (for savefig.format).